interface/portraits/portraits_highelf.gfx

```diff
@@ -221,14 +221,14 @@ spriteTypes = {
 
 			"GFX_highelf_female_base:p2"
```

Comment:
You need to add:
- [ ] `GFX_highelf_female_clothes_behind`
- [ ] `GFX_highelf_female_headgear_behind`
- [ ] `GFX_highelf_female_hair_behind`
- [ ] `GFX_highelf_female_headgear_mid`
- [ ] `GFX_highelf_female_clothes_infront`

May be empty, but the files must exist. Without them, hairs/clothes don't work correctly. For example, you can't change hair in-game.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81166083-2ec08980-8fa4-11ea-918e-992e773c6e6d.png)

</details>